byterun/win32.c

```diff
@@ -57,10 +57,23 @@
 #endif
 
 /* Very old Microsoft headers don't include intptr_t */
-#if defined(_MSC_VER) && !defined(_UINTPTR_T_DEFINED)
+#ifdef _MSC_VER
+#ifndef _UINTPTR_T_DEFINED
+#ifdef _WIN64
+typedef unsigned long int uintptr_t;
```

Comment:
This is wrong; `long` is 32-bit on Windows, see https://msdn.microsoft.com/en-us/library/3b2e7499.aspx.